doc/node.1

```diff
@@ -119,6 +119,12 @@ Enable experimental JSON interop support for the ES Module loader.
 .It Fl -experimental-modules
 Enable experimental ES module support and caching modules.
 .
+.It Fl -experimental-require-target
+Enable experimental support for a "require" conditional export.
+.
+.It Fl -experimental-resolve-self
```

Comment:
should this be added in a separate pr?

Reply:
In theory, yes, I was bug fixing for @jkrems while I was there.

Reply:
just thinking about backporting later. Probably better to separate (even a different commit in this PR)

Reply:
I've separated this out as a doc commit.